src/test/java/uk/sky/cqlmigrate/SchemaUpdatesTest.java

```diff
@@ -59,32 +51,28 @@ public void tearDown() {
     }
 
     @AfterClass
-    public static void tearDownClass() throws Exception {
+    public static void tearDownClass() {
         EmbeddedCassandraServerHelper.cleanEmbeddedCassandra();
         Uninterruptibles.sleepUninterruptibly(100, TimeUnit.MILLISECONDS);
     }
 
     @Test
-    public void schemaUpdatesTableShouldBeCreatedIfNotExists() throws Exception {
+    public void schemaUpdatesTableShouldBeCreatedIfNotExists() {
         //given
-        cluster.connect("system").execute("CREATE KEYSPACE " + TEST_KEYSPACE + " WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1 };");
-        Session session = cluster.connect(TEST_KEYSPACE);
         SessionContext sessionContext = new SessionContext(session, ConsistencyLevel.ALL, ConsistencyLevel.ALL, clusterHealth);
         SchemaUpdates schemaUpdates = new SchemaUpdates(sessionContext, TEST_KEYSPACE);
 
         //when
         schemaUpdates.initialise();
 
         //then
-        KeyspaceMetadata keyspaceMetadata = cluster.getMetadata().getKeyspace(TEST_KEYSPACE);
+        KeyspaceMetadata keyspaceMetadata = session.getMetadata().getKeyspace(TEST_KEYSPACE).get();
         assertThat(keyspaceMetadata.getTable(SCHEMA_UPDATES_TABLE)).as("table should have been created").isNotNull();
```

Comment:
can check optional is empty? if the test fails the .get() with throw an exception rather than fail this assertion.